src/main/java/pl/techlab24/OSKManager/model/CourseClient.java

```diff
@@ -35,6 +35,9 @@
     @OneToMany
     private List<Transaction> transactions;
 
+    @OneToMany
+    private List<RideDetails> rideDetails;
```

Comment:
in terms of conversation with @rdx7777 it should be here

Reply:
We decided to use bidirectional relationship between classes, so if in `RideDetails` class we have `CourseClient courseClient` field with `@ManyToOne` annotation, we have to add bidirectional relationship in `CourseClient` class:
```
@OneToMany
private List<RideDetails> rideDetails;
```

Reply:
I do not claim that this bidirectional relation shouldn't be in code at all, I only claim that it is part of another issue (#8), and this issue may be completed without code from issue #9. I prepared a small demo: a branch auxilary, see: https://github.com/JavaAppforFuture/oskmanager-backend/tree/auxiliary
You may see there two commits: the first contains code for #8 without any code from issue #9. It works correctly in my opinion :)

Reply:
Well, I didn't changed my mind and still claim that this is out of the scope of this issue. If you allow devs not to stick to their issues, it will cause a complete mess. But if it's OK for you, there is nothing else for me but accept it ;-)

Reply:
As I wrote on Discord - it was my fault during tasks writing :-)
Next time I will be more accurate, I promise :-)
Now I propose to finish this pull request. Is it ok for you?